### PR TITLE
Link Browser Pages: document configuration option pageIdSelector

### PR DESCRIPTION
**Pull request by josefglatz**

### CODE REVIEW

**Comment by josefglatz** [comment]
Related Gerrit Patches are:

* https://review.typo3.org/c/Packages/TYPO3.CMS/+/64065
* https://review.typo3.org/c/Packages/TYPO3.CMS/+/64064

File: Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst [hunk 1]
```diff
@@ -124,6 +124,25 @@ The PageTSconfig of the LinkHandler is being used in sysext `recordlist`
 in class :php:`\TYPO3\CMS\Recordlist\LinkHandler\RecordLinkHandler`
 which does not contain Hooks or Slots.
 
+Special: Activate page UID field for directly entering a target page
```
Comment (georgringer) [inline Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst:127]: I find the word "special" strange. can we just remove it?

File: Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst [hunk 2]
```diff
@@ -124,6 +124,25 @@ The PageTSconfig of the LinkHandler is being used in sysext `recordlist`
...
+--------------------------------------------------------------------
+
+.. important::
+
+   This only works for the page LinkHandler. Not for custom added LinkHandler configurations.
```
Comment (andreasfernandez) [inline Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst:132]: "PageLinkHandler"

File: Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst [hunk 3]
```diff
@@ -124,6 +124,25 @@ The PageTSconfig of the LinkHandler is being used in sysext `recordlist`
...
+
+It is possible to activate an additional field in the link browser to enter the UID of a page
```
Comment (andreasfernandez) [inline Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst:134]: possible to enable an additional

File: Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst [hunk 4]
```diff
@@ -124,6 +124,25 @@ The PageTSconfig of the LinkHandler is being used in sysext `recordlist`
...
+directly instead of selecting it from the page tree. This is especially useful if you have a
+potentially large page tree in your TYPO3 project.
+
+.. figure:: ../../../Images/LinkBrowserTSConfigExamplepageIdSelector.png
+   :alt: The link browser field for entering a page uid
+
+Activate the field with exactly the following User-/PageTSConfig::
```
Comment (andreasfernandez) [inline Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst:141]: Enable the field

File: Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst [hunk 5]
```diff
@@ -124,6 +124,25 @@ The PageTSconfig of the LinkHandler is being used in sysext `recordlist`
...
+Special: Activate page UID field for directly entering a target page
+--------------------------------------------------------------------
+
+.. important::
+
+   This only works for the page LinkHandler. Not for custom added LinkHandler configurations.
+
+It is possible to activate an additional field in the link browser to enter the UID of a page
+directly instead of selecting it from the page tree. This is especially useful if you have a
+potentially large page tree in your TYPO3 project.
```
Comment (georgringer) [inline Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst:136]:
```suggestion
potentially large page tree in your TYPO3 project and you know the the id of the to be linked page already.
```

File: Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst (hunk 2 above)
Comment (georgringer) [inline Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst:132]:
```suggestion
   This only works for the page LinkHandler and **not** for custom added LinkHandler configurations.
```

File: Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst (hunk 4 above)
Comment (georgringer) [inline Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst:141]:
```suggestion
Activate the field with the following User-/PageTSConfig:
.. code-block:: typoscript

```

**Comment by DanielSiepmann** [comment]
I would like to have it in TSconfig, and just add a reference to their.
Also some more examples would be cool, e.g. how to add to link wizard in bodytext when using CKeditor?

**Comment by josefglatz** [comment]
> I would like to have it in TSconfig, and just add a reference to their.

K, have it on my todo list

> Also some more examples would be cool, e.g. how to add to link wizard in bodytext when using CKeditor?

@DanielSiepmann Do you have a working example? I know we talked about, but I forgot absolutely where to search for and what ಠ_ಠ

**Comment by DanielSiepmann** [comment]
I've adjusted accordingly to comments.

Let's add it here for now. We might cleanup things later, or not.

I also don't have examples. So let's merge to improve the existing part.

**Pull request merged**